Update Google I/O Extended

jigyasa-grover opened the pull request.

jigyasa-grover committed:
Update google_io_extended.md

jigyasa-grover committed:
Update google_io_extended.md

friedger commented on gdg_events/google_io_extended.md:
```diff
@@ -2,6 +2,16 @@
+
+To overcome this, Google has created an initiative which helps extending this conference through various communities like the Google Developer Groups, Google Business Groups etc, titled as `Google I/O Extended` to different locations through interested organizers for the benefit of developers, students, start-up junkies around the world.
+
+[Register your Google I/O Extended 2015 event](https://events.google.com/io2015/form)
```
This is outdated.

friedger commented on gdg_events/google_io_extended.md:
```diff
@@ -2,6 +2,16 @@
+
+Google I/O features highly technical, in-depth sessions focused on building web, mobile, and enterprise applications with Google and open web technologies such as Android, Chrome, Chrome OS, Google APIs, Google Web Toolkit, App Engine, and more.
+
+As interesting as it sounds, many developers around the world feel left out as they are unable to attend the Google I/O event every year. Students and Industry Experts are always vying to get a glimpse of this much awaited event, but in vain.
```
This is too opinionated, for my taste. Could you reformulate in a more objective language?

webgrrrl replied:
For lines 9 and 10, possibly combine them to sound as follows:
+To extend this to developers who are unable to attend the event on-site, and reach out to various communities like the Google Developer Groups, Google Business Groups, and others, an event known as `Google I/O Extended` is organized by local Google community groups around the world. I/O Extended events can be run in tandem with the actual I/O event, or can be run based on the timing that each local community feels suitable.

friedger commented on gdg_events/google_io_extended.md:
```diff
@@ -2,6 +2,16 @@
+
+As interesting as it sounds, many developers around the world feel left out as they are unable to attend the Google I/O event every year. Students and Industry Experts are always vying to get a glimpse of this much awaited event, but in vain.
+
+To overcome this, Google has created an initiative which helps extending this conference through various communities like the Google Developer Groups, Google Business Groups etc, titled as `Google I/O Extended` to different locations through interested organizers for the benefit of developers, students, start-up junkies around the world.
```
Could you extend this section and add what is Google offering and what is the organizers providing? What is needed to organize an IO extended?

webgrrrl replied:
A summary of the [organiser guide](https://docs.google.com/presentation/d/1rUwLzz3-Hgcs9eu1uph4En1FUCcpTQspw-xpK4Kaoms/pub) and then a link to it would suffice I think.

friedger commented:
@jig08 Are you still interested in working on this section? If not could you please close the pull request?

jigyasa-grover commented:
@friedger Yes :)
Shall do the required changes and send a PR soon.
PS: Having university exams this week

jigyasa-grover committed:
Update google_io_extended.md

- Put up the content in a more objective language.
- Added link to the Organizer's Guide & Removed the previous outdated 2015 link.

jigyasa-grover commented:
- Have put up the content in a more objective language.
- Added link to the Organizer's Guide & Removed the previous outdated 2015 link.

jigyasa-grover commented:
@friedger Kindly review this PR :)

friedger commented:
Nice

jigyasa-grover commented:
@friedger Thank You :)
